cmake/modules/FindTensorFlow.cmake

```diff
@@ -196,9 +214,41 @@ if(${TF_DISABLE_ASSERTS})
   add_definitions("-DNDEBUG=True")
 endif()
 
+if(IS_DIRECTORY $ENV{TENSORFLOW_BUILD_DIR})
```

Comment:
Can `IS_DIRECTORY` be used with  CMAKE 2.8 versions? If not, can we test the CMake version in `FindTensorflow.cmake`?

Reply:
Another issue I was going to propose is listed as an issue in the Github issue tracker.  The QT, ITK, VTK, Slicer, and many other CMake based communities have moved to require newer (say 3.8 or newer) versions of CMake.  CMake < 3.0 does not support for C++11 in any intelligent way.  At least CMake 3.8 is recommended, and advanced C++11 projects should probably use CMake 3.12 or newer to get all the language features exposed well.

Reply:
Switching to Cmake 3.8 would take some time as I need to rebuild all Docker containers. As Ubuntu 16.04 ships Cmake version 3.5, I guess 3.5 would be a safe option.

Reply:
For this module, 3.5 would currently be sufficient for all the features that we are currently using.  It would only be for advanced C++ introspection features that 3.12 is needed for, and we are not using those.

Thank you for the feedback.